modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlertActivity.java

```diff
@@ -32,6 +40,13 @@
 	private String message;
 	private Button btnOK;
 	private TextView txtMessage;
+	private Uri defaultRingtoneUri;
+	private Ringtone defaultRingtone;
+	private DeviceInfo deviceInfo;
+	private String type;
+
+	private static final String RING = "ring";
```

Comment:
It would have been ideal if the constant was made something like "OPERATION_RING". Then it is clear to the one who reads the code that it is a operation.